.circleci/config.yml

```diff
@@ -2351,18 +2301,12 @@ workflows:
  smoke-tests:
    <<: *skip_release_build_branch_push
    jobs:
-     - smoke-standalone-26:
-         context: scalyr-agent
      - smoke-standalone-27:
          context: scalyr-agent
      - smoke-standalone-35:
          context: scalyr-agent
      - smoke-standalone-35-rate-limit:
          context: scalyr-agent
-     - smoke-standalone-26-tls12:
-         context: scalyr-agent
-     - smoke-standalone-27-tls12:
```

Comment:
Should the 2.7 version also be removed?

Reply:
Sadly we can't do that just yet - we will be able to do that once we actually officially remove support for it.

Reply:
I mean particularly the `smoke-standalone-27-tls12` :)